server/py/services/api/api/endpoints/nuclio.py

```diff
@@ -216,11 +220,28 @@ async def delete_api_gateway(
         return await client.delete_api_gateway(project_name=project, name=name)
 
 
-@router.post("/projects/{project}/nuclio/{name}/deploy")
+@router.post(
+    "/projects/{project}/nuclio/{name}/deploy",
+    response_model=DeployResponse,
+    responses={
```

Comment:
since you are updating the deploy API, I recommend checking with someone from the platform team to verify that this response structure is valid